### ISSUE TITLE
Control color of all arcs on hover

### ISSUE THREAD

**Issue by patryk-wlaz** [vasturiano/sunburst-chart#51]
**Describe the solution you'd like**
I would like to be able to control color of all arcs on hover. E.g. make all arc other than hovered whitened, and change the color of hovered arc at the same time.

**Additional context**
Thanks to reaching out to @vasturiano I know that we can do something like this:
`myChart.onHover(hoveredArc => myChart.color(arc => arc === hoveredArc ? 'purple' : 'white'));` 

I have a follow up question - is there anything we can do to change color immediately duringHover, without default opacity?

I tried overwriting css classes like this (scss, they get assigned properly - I verified this in devtools):
`
.sunburst-viz {
  .main-arc {
    transition: none !important;

    &:hover {
      transition: none !important;
    }
  }
}
`
It doesn't seem to change anything.

**Comment by vasturiano** [comment]
@patryk-wlaz thanks for logging your case.

As for your follow-up question, the CSS that's causing the opacity transition is here:
https://github.com/vasturiano/sunburst-chart/blob/b12e607d0f6899d4d57098b2be288b14da60bad8/src/sunburst.css#L27-L30

You should be able to override those style on your application. Do you wish to cancel the opacity effect on hover, or just make the opacity transition immediate?

**Comment by patryk-wlaz** [comment]
I wish to cancel opacity and make color change immediate. Thank you for pointing out styles, but I've alread found them. I've managed to target them by scss mentioned before. This is what I see in developer tools:
![image](https://user-images.githubusercontent.com/15091443/83998659-5f0f9500-a961-11ea-9194-2db3cf75627b.png)
On hover: 
![image](https://user-images.githubusercontent.com/15091443/83998813-b4e43d00-a961-11ea-8910-fa65ba60bf73.png)

I'm starting to fell that I do some silly mistake here, like overwriting css styles in a wrong way :) But I think forcing opacity to be one and transition to be none should work...

**Comment by patryk-wlaz** [comment]
Yeah I thought so, but I addd transition just to be sure. As you see, it doesn't work - I don't know what am I doing wrong here [see video of my screen captured](https://pwlaz.tinytake.com/tt/NDM1OTUwOV8xMzc1MDcxMg)
